examples/blob/main.py

```diff
@@ -0,0 +1,42 @@
+from pyteal import (
```

Comment:
I needed to add an empty `__init__.py` under `examples/` and used the following `launch.json` in order to get VS code to visually debug `blob/main.py`, `iter/main.py` as well as `iter_test.py`:

* `launch.json`:

```json
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "blob example",
            "type": "python",
            "request": "launch",
            "module": "examples.blob.main",
            "console": "integratedTerminal"
        },
        {
            "name": "iter example",
            "type": "python",
            "request": "launch",
            "module": "examples.iter.main",
            "console": "integratedTerminal"
        },
        {
            "name": "Python: Current File",
            "type": "python",
            "request": "launch",
            "program": "${file}",
            "console": "integratedTerminal"
        }
    ]
}
```

Reply:
I've been just running `python main.py` in the cli and it may be just finding the locally installed version in my venv.  This is probably bad practice so I'll need to look into this